tools/bin/commands/integration-test

```diff
@@ -37,6 +42,29 @@ maven_args() {
         args="$args -Dsyndesis.version=${release_version}"
     fi
 
+    local image_tag="$(readopt --image)"
+    if [ -n "${image_tag}" ]; then
+        args="$args -Dsyndesis.image.tag=${image_tag}"
+    fi
+
+    if [ "$(hasflag --logging)" ]; then
+        args="$args -Dsyndesis.logging.enabled=true"
```

Comment:
Shouldn't this refer to the property ${syndesis.logging.enabled}?

Reply:
No not really as this is a shortcut flag on the Syndesis cli tool and it overwrites the property used in Maven